docs/team/contributors-binder.yaml

```diff
@@ -4,65 +4,76 @@
   affiliation: UC Berkeley
   contributions: doc
   team: red
-  
+  last-check-in: 2018-11
```

Comment:
How did you decide the dates? I was expecting to see everyone start out with 2019-07 because we have "no records" from before. It would have the nice side effect of having everyone "in sync" so we don't have to check every month (it will drift but we could start with an easy setup :) )

Reply:
Makes sense - I'll update the dates to start on the same month